src-backbone/gulpfile.js

```diff
@@ -79,56 +79,57 @@ gulp.task('css', function() {
 //------------------------------------------------------------------------------
 
 gulp.task('js-lint', function() {
-    var jshint = require('gulp-jshint'); // Note: we're using JSHint, a better fork of the original JSLint
-    var stylish = require('jshint-stylish');
-    var filter = require('gulp-filter');
-
-    var js_filter = filter('**/*.js');
+    let jshint = require('gulp-jshint'); // Note: we're using JSHint, a better fork of the original JSLint
+    let stylish = require('jshint-stylish');
+    let filter = require('gulp-filter');
 
     gulp.src(Config.javascripts)
-        .pipe(js_filter)
+        .pipe(filter('**/*.js'))
```

Comment:
Wanted to clean up simple variables. In hindsight, this was a bad idea as I need to add this back because the filter needs more rules in another commit